### DISCUSSION TITLE
Api widget

### DISCUSSION THREAD

**Discussion by danz0l99** [glanceapp/glance#153] [General]
Would be really handy if we could have an API widget to display data from remote sources using endpoints.

So say the amount of recipes, users, cookbooks you have for instance from a mealie server using a token and endpoint.

Something like (mealie example)

          - type: api
            url: "http://localhost/api/admin/about/statistics"
            token: "YOUR API TOKEN"
            cache: 1s
            parameters:
              {1} - Total Recipes # first json item
              {2} - Total Users: # second json item
              {3} - Total Groups # third json response

etc ? not sure how achievable this is but an API widget would be very powerful to bring in outside information to the dashboard.

**Comment by paulkakell** [comment]
Would really like this as well.  I'm looking at creating a dashboard for my sales through Sellbrite/eBay and it would be super helpful.
